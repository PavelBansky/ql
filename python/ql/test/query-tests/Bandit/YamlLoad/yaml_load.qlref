Bandit/yaml_load.ql