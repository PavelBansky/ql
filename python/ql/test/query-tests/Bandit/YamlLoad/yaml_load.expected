| Test.py:7:9:7:17 | Attribute |  Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load(). |
| Test.py:10:13:10:21 | Attribute |  Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load(). |
| Test.py:13:13:13:21 | Attribute |  Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load(). |
